packages/blockchain-wallet-v4-frontend/src/scenes/SecurityCenter/BasicSecurity/TwoStepVerification/GoogleAuth/index.js

```diff
@@ -38,6 +47,15 @@ class GoogleAuthContainer extends React.PureComponent {
     this.props.modalActions.showModal('TWO_STEP_SETUP_MODAL')
   }
 
+  handleNotification() {
+    const { alertActions } = this.props
+    this.setState({ notificationActive: true })
+    this.timeout = setTimeout(() => {
+      this.setState({ notificationActive: false })
+    }, 2000)
```

Comment:
doesnt the alert auto-dismiss? why do we need another piece of state? implementation is below

```
const displaySuccess = (message: string, data?: any, persist?: boolean) =>
  display('success', message, data, persist)
```

Reply:
This is now cleaned up.